modules/machine-user-infra-machines-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z-kvm.adoc
+
+:_content-type: PROCEDURE
+[id="machine-user-infra-machines-ibm-z-kvm_{context}"]
+= Creating {op-system} machines using virt-install
+
+You can create more {op-system-first} compute machines for your {ibmzProductName} cluster by using virt-install.
+
+.Prerequisites
+
+* At least one LPAR running on {op-system-base} 8.7 or later with KVM, referred to as {op-system-base} KVM host in this procedure.
+* The KVM/QEMU hypervisor is installed on the {op-system-base} KVM host.
+* A domain name server (DNS) that can perform hostname and reverse lookup for the nodes.
+* An HTTP or HTTPS server is set up.
+
+.Procedure
+
+. Extract the Ignition config file from the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc extract -n openshift-machine-api secret/worker-user-data-managed --keys=userData --to=- > worker.ign
+----
+
+. Upload the `worker.ign` Ignition config file you exported from your cluster to your HTTP server. Note the URL of this file.
+
+. You can validate that the Ignition file is available on the URLs. The following example gets the Ignition config file for the compute node:
++
+[source,terminal]
+----
+$ curl -k http://<HTTP_server>/worker.ign
+----
+
+. Download the {op-system-base} live kernel, initramfs, and rootfs files by running to following commands:
++
+[source,terminal]
+----
+curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.kernel.location')
+----
++
+[source,terminal]
+----
+curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
```

Comment:
should have a $ in front (if this is the consistent style of writing commands executed on a shell)